.ci/install_qemu.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -e
+
+QEMU_REPO="github.com/kata-containers/qemu"
+QEMU_DIR="qemu"
+KATA_QEMU_BRANCH="stable-2.11"
+
+git clone "https://${QEMU_REPO}"
+
+pushd "$QEMU_DIR"
+git checkout "$KATA_QEMU_BRANCH"
+
+echo "Build Qemu"
+./configure.sh
```

Comment:
I see that we have an old version of the configuration script in the qemu branch.

But as noted in https://github.com/kata-containers/linux/pull/5#issuecomment-372665615, we should be using the script from the packaging repo, which will be:

- https://github.com/kata-containers/packaging/issues/7

Reply:
Opened https://github.com/kata-containers/packaging/pull/10 to address this.